### ISSUE TITLE
Disenchanting with full inventory

### ISSUE THREAD

**Issue by dhende917** [elysium-project/server#2328]
Apparently this is an old bug from vanilla times.  I recently disenchated a bow and was to receive 2 lesser magic essences, however ny inventory was "full." Obvously once I would disenchant the bow, I would have space for something new.  So I sit here looking at 2 lesser magic essences in a loot window with 1 open space in my inventory, but I cannot loot the window.  Thank you for your time.

**Comment by berillis** [comment]
Yes, this has happened the whole time the server (Elysium) has been up. I suppose the fix should be about the same as for the healthstones?

**Comment by Sirael11** [comment]
Isn't this the same way that quest rewards also work? Like when you turn in a quest with 2 quest items while having 0 inventory space and get 1 reward then it doesn't work. You have to delete an item for it to work and end up with 2 free spaces. IIRC that's how it worked back in vanilla though, so not sure if it will be changed.
